crates/components/src/lib.rs

```diff
@@ -84,6 +83,36 @@ impl<'e, T: ToString + PartialEq + strum::IntoEnumIterator> egui::Widget for Enu
     }
 }
 
+pub struct EnumRadioList<'e, T> {
+    current_value: &'e mut T,
+}
+
+impl<'e, T> EnumRadioList<'e, T> {
+    pub fn new(current_value: &'e mut T) -> Self {
+        Self { current_value }
+    }
+}
+
+impl<'e, T: ToString + PartialEq + strum::IntoEnumIterator> egui::Widget for EnumRadioList<'e, T> {
+    fn ui(self, ui: &mut egui::Ui) -> egui::Response {
+        let mut changed = false;
+        let mut response = ui
+            .vertical(|ui| {
+                for variant in T::iter() {
+                    let text = variant.to_string();
+                    if ui.radio_value(self.current_value, variant, text).changed() {
+                        changed = true;
+                    }
+                }
+            })
+            .response;
+        if changed {
+            response.mark_changed();
+        }
+        response
+    }
+}
```

Comment:
Why does the left "Options" column of the event editor window expand when the event editor window is resized horizontally while the right "Trigger" column stays the same size? I think the right column should also expand when the window is resized for consistency. One way to fix this would be to use `ui.with_cross_justify` (defined in crates/components/src/ui_ext.rs) inside of the `EnumRadioList` widget.

```suggestion
impl<'e, T: ToString + PartialEq + strum::IntoEnumIterator> egui::Widget for EnumRadioList<'e, T> {
    fn ui(self, ui: &mut egui::Ui) -> egui::Response {
        let mut changed = false;
        let mut response = ui
            .vertical(|ui| {
                ui.with_cross_justify(|ui| {
                    for variant in T::iter() {
                        let text = variant.to_string();
                        if ui.radio_value(self.current_value, variant, text).changed() {
                            changed = true;
                        }
                    }
                });
            })
            .response;
        if changed {
            response.mark_changed();
        }
        response
    }
}
```